pandas/tests/io/parser/usecols.py

```diff
@@ -492,11 +492,11 @@ def test_raise_on_usecols_names_mismatch(self):
         tm.assert_frame_equal(df, expected)
 
         usecols = ['a', 'b', 'c', 'f']
-        with tm.assert_raises_regex(ValueError, msg):
+        with tm.assert_raises_regex(ValueError, msg.format(missing="\['f'\]")):
```

Comment:
can you add a test where you have 2 missing cols?

Reply:
Yep, will do this and add the whatsnew entry tomorrow, thanks!